CMakeLists.txt

```diff
@@ -508,8 +508,8 @@ ADD_DEFINITIONS(-DDATADIR=\\"${DATADIR}\\")
 SET (MAN_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/share/man")
 
 # include custom opencog cmake macros
-INCLUDE("${COGUTIL_DATA_DIR}/cmake/OpenCogMacros.cmake")
-INCLUDE("${COGUTIL_DATA_DIR}/cmake/OpenCogFunctions.cmake")
+INCLUDE("${DATADIR}/cmake/OpenCogMacros.cmake")
```

Comment:
```COGUTIL_DATA_DIR``` and ```DATADIR``` can be different.
```COGUTIL_DATA_DIR``` is a place where ```cogutil``` module is installed.
```DATADIR``` in ```opencog``` is a place where opencog will be installed.
```OpenCogMacros.cmake``` is located in ```cogutil``` module so ```COGUTIL_DATA_DIR``` is correct to use here.